media-sound/clementine/clementine-1.4.0_rc2-r2.ebuild

```diff
@@ -14,9 +14,9 @@ if [[ ${PV} == *9999* ]]; then
 	EGIT_REPO_URI="https://github.com/clementine-player/Clementine.git"
 	inherit git-r3
 else
-	S="${WORKDIR}/Clementine-${PV/_}"
 	SRC_URI="https://github.com/clementine-player/Clementine/archive/refs/tags/${PV/_}.tar.gz -> ${P}.tar.gz"
 	KEYWORDS="~amd64 ~arm64 ~ppc64 ~x86"
+	S="${WORKDIR}/Clementine-${PV/_}"
```

Comment:
Sorry if I sound nitpicky, the proper order is `SRC_URI`, then `S` and then `KEYWORDS`.  You can see it in `skel.ebuild`.